### DISCUSSION TITLE
Action not triggered, neither on pull request, nor on push

### DISCUSSION THREAD

**Discussion by iamareebjamal** [community/community#26572] [Actions]
<p><a href="https://github.com/fossasia/open-event-server/pull/6476" rel="nofollow noopener">https://github.com/fossasia/open-event-server/pull/6476</a></p>
<p>Added an action and pushed to a branch and created a pull request. No action got triggered.</p>
<p>Does the name of the workflow YML have any role in the process? I suppose not. </p>
<p>However, I’ll try to merge it in the default branch and see if it changes anything. However, I tried just creating a PR in private repo 3 days ago and it worked without the workflow committed to the default branch.</p>
<p>Also, when I go to checks panel of the pull request, it just hangs on Waiting for checks information…</p>
<p><a href="https://github.com/fossasia/open-event-server/pull/6476/checks" rel="nofollow noopener">https://github.com/fossasia/open-event-server/pull/6476/checks</a></p>

**Comment by iamareebjamal** [comment]
<p>Still, no trigger <a href="https://github.com/fossasia/open-event-server/actions" rel="nofollow noopener">https://github.com/fossasia/open-event-server/actions</a> When I went here, it said “Re-enable actions for this repository”, “e had problems enabling actions for this repo … normally this can be fixed by just re-enabling”</p>
<p>I re-enabled them and updated a few PRs, but no action has been triggered</p>

**Comment by iamareebjamal** [comment, marked as answer]
<p>Seems like Actions were disabled in organization. So, I had to enable them in organization settings and then enable it in repository settings. </p>
<p>I think it should be apparent to the user and shown that actions are not enabled for the repository or organization and at which level</p>

**Comment by chrispat** [comment]
<p>When actions are disabled for an organization or repo the Actions tab should be hidden the repo.  Are you still seeing the Actions tab and workflows are not running?</p>

**Comment by iamareebjamal** [comment]
<p>No, the action tab was hidden, but I thought it was a bug in the extension I was using, so I appended /actions in the repo URL to access the section</p>

**Comment by IgorGanapolsky** [comment]
<p>You have to actually click on the <strong>Actions</strong> tab in order to explicitly see the executing Action.  Otherwise, the Pull Request screen doesn’t always show them.</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/9/091ff5236701f018725fbbec8fcd962f0df7926e.png" data-download-href="/uploads/short-url/1iILuyQojbZJLPV3tGEn0KO3py6.png?dl=1" title="Screen Shot 2021-04-28 at 12.18.25 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/201209/181098527-b3eaa90c-098d-481d-99ce-89193d1fbcea.png" alt="Screen Shot 2021-04-28 at 12.18.25 PM" data-base62-sha1="1iILuyQojbZJLPV3tGEn0KO3py6" width="690" height="52" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/9/091ff5236701f018725fbbec8fcd962f0df7926e_2_690x52.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/9/091ff5236701f018725fbbec8fcd962f0df7926e_2_1035x78.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/9/091ff5236701f018725fbbec8fcd962f0df7926e_2_1380x104.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/9/091ff5236701f018725fbbec8fcd962f0df7926e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-04-28 at 12.18.25 PM</span><span class="informations">2274×174 15.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
